scripts/exwave_init.sh

```diff
@@ -129,7 +129,11 @@
       fi
 
       [[ ! -d $COMOUT/rundata ]] && mkdir -m 775 -p $COMOUT/rundata
-      echo "$USHwave/wave_grid_moddef.sh $grdID > $grdID.out 2>&1" >> cmdfile
+      if [ ${CFP_MP:-"NO"} = "YES" ]; then
+        echo "$nmoddef $USHwave/wave_grid_moddef.sh $grdID > $grdID.out 2>&1" >> cmdfile
```

Comment:
Are you sure this will work on WCOSS ?    CFP on wcoss should not have $nmoddef included in the cmdfile.

Reply:
OK, I understand the logic now.  Please ignore my comment here